### DISCUSSION TITLE
using Reticulum to facilitate Shamir's Secret Sharing

### DISCUSSION THREAD

**Discussion by trosel** [markqvist/Reticulum#67] [Ideas]
See [SLIP-0039](https://github.com/satoshilabs/slips/blob/master/slip-0039.md) for more info. Also see [Argent's "Guardian" system](https://support.argent.xyz/hc/en-us/articles/360022631992-About-Guardians).

This can obviously be used for *many* things. But mostly I think it gives people more peace of mind when doing self-custody of their valuable digital goods.

But SSS needs a network for communication to take place. Reticulum can of course provide that.

**Comment by markqvist** [comment]
This is a really interesting proposal, and Reticulum would be well suited to this. Unfortunately I don't have time to work on this myself, but if anyone does, I will continue to monitor this thread and answer any questions (if I can).
